src/components/Header/utils/links.tsx

```diff
@@ -0,0 +1,33 @@
+import { HeaderType } from './types'
```

Comment:
should be ts file.
shouldn't be under utils folder. maybe call the folder links or something